crates/chain/src/chain_graph.rs

```diff
@@ -151,7 +151,7 @@ where
                         let _ = inflated_chain
                             .insert_tx(*txid, pos.clone())
                             .expect("must insert since this was already in update");
-                        let _ = inflated_graph.insert_tx(tx);
+                        let _ = inflated_graph.insert_tx(tx.clone());
```

Comment:
Seems okay not to clone here? It's already an owned `tx`.

Reply:
In my mind this file should have no changes whatsoever.

Reply:
The change was probably accidental.